mmv1/templates/terraform/examples/alloydb_cluster_basic.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_alloydb_cluster" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  cluster_id = "<%= ctx[:vars]['alloydb_cluster_name'] %>"
+  location   = "us-central1"
+  network    = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
+
+  initial_user {
+    password = "<%= ctx[:vars]['alloydb_cluster_name'] %>"
+  }
```

Comment:
recommend flexing all resource fields that are set-able. If not it will incur more review passes as I will be evaluating this a future pass.

Reply:
+1 this goes back to https://github.com/GoogleCloudPlatform/magic-modules/pull/6208#pullrequestreview-1026432985 - need to have a "full" test in addition to basic and update.

Reply:
just making sure i understand, you are asking here for a test with all fields that are set-able. thats what you mean by "flexing".

Reply:
yes, that's correct - a test that sets values for as many of the fields as possible.

Reply:
Added